cmd/malwareScan.go

```diff
@@ -0,0 +1,166 @@
+package cmd
+
+import (
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+	"github.com/SAP/jenkins-library/pkg/command"
+	piperhttp "github.com/SAP/jenkins-library/pkg/http"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+)
+
+var open = _open
+var getSHA256 = _getSHA256
+
+func _open(path string) (io.ReadCloser, error) {
+	return os.Open(path)
```

Comment:
os.Open is `func(name string) (*os.File, error)`. When we define and assign here in one step `_open` is also `func(name string)(* os.File, error)`. That is not exactly what I need (especially when we take the tests into account). There we need `func(name string) (io.ReadCloser, error) since a read closer is - to my understanding - easier to mock.That is the reason why I did not assign that directly.

In case somebody manages it to get that inlined so that we don't mess up with `io.ReadCloser` and `*os.File`, go ahead, I'm eager to learn ...